### PR TITLE
docs: update links

### PR DESCRIPTION
**Pull request by Wwwsylvia**
Resolves: #554

### CODE REVIEW

File: registry/remote/repository.go [hunk 1]
```diff
@@ -1106,7 +1106,7 @@ func (s *manifestStore) deleteWithIndexing(ctx context.Context, target ocispec.D
 //
 // References:
 //   - https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc3/spec.md#deleting-manifests
-//   - https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#deleting-manifests
+//   - https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc3/spec.md#deleting-manifests
```
Comment (TerryHowe) [inline registry/remote/repository.go:1109]:
```suggestion
```

File: registry/remote/repository.go [hunk 2]
```diff
@@ -1335,7 +1335,7 @@ func (s *manifestStore) pushWithIndexing(ctx context.Context, expected ocispec.D
 //
 // References:
 //   - https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc3/spec.md#pushing-manifests-with-subject
-//   - https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#pushing-manifests-with-subject
+//   - https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc3/spec.md#pushing-manifests-with-subject
```
Comment (TerryHowe) [inline registry/remote/repository.go:1338]:
```suggestion
```

File: registry/remote/repository.go [hunk 3]
```diff
@@ -528,7 +528,7 @@ func (r *Repository) referrersPageByAPI(ctx context.Context, artifactType string
 	if artifactType != "" {
 		// check both filters header and filters annotations for compatibility
 		// reference for filters header: https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc3/spec.md#listing-referrers
-		// reference for filters annotations: https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#listing-referrers
+		// reference for filters annotations: https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc3/spec.md#listing-referrers
```
Comment (TerryHowe) [inline registry/remote/repository.go:531]:
```suggestion
```

File: registry/remote/repository.go [hunk 4]
```diff
@@ -76,7 +76,7 @@ const (
 //
 // References:
 //   - https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc3/spec.md#listing-referrers
-//   - https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#listing-referrers
+//   - https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc3/spec.md#listing-referrers
```
Comment (TerryHowe) [inline registry/remote/repository.go:79]:
```suggestion
```

File: registry/remote/repository.go (hunk 1 above)
Comment (Wwwsylvia) [reply registry/remote/repository.go]: Oh this was a bug, thanks for catching it! Both versions of specs are supported so they are listed here.

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**